src/translib/transformer/xfmr_system_aaa.go

```diff
@@ -42,8 +42,7 @@ func init () {
 
 var YangToDb_auth_method_xfmr FieldXfmrYangToDb = func(inParams XfmrParams) (map[string]string, error) {
     if log.V(3) {
-        log.Info( "YangToDb_auth_method_xfmr: root: ", inParams.ygRoot,
-            ", uri: ", inParams.uri, "param: ", inParams.param)
+	    xfmrLogInfoAll( "YangToDb_auth_method_xfmr: root: %v, uri: %v param: %v", inParams.ygRoot, inParams.uri, inParams.param)
```

Comment:
The change to this file can be reverted. It is already within if log.V(3) check.